samples/cellular/nrf_cloud_mqtt_multi_service/src/shadow_support_coap.c

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/logging/log_ctrl.h>
+#include <modem/location.h>
+#include <date_time.h>
+#include <stdio.h>
+#include <net/nrf_cloud.h>
+#include <net/nrf_cloud_coap.h>
+
+#include "shadow_support_coap.h"
+
+LOG_MODULE_REGISTER(shadow_support_coap, CONFIG_MULTI_SERVICE_LOG_LEVEL);
+
+#define COAP_SHADOW_MAX_SIZE 512
+
+static int check_shadow(void)
+{
+	int err;
+	char buf[COAP_SHADOW_MAX_SIZE];
+
+	buf[0] = '\0';
+	LOG_INF("Checking for shadow delta...");
```

Comment:
i think all the LOG_INFs should be LOG_DBGs.

Reply:
I want to keep the inf for the delta itself, if one is received.